src/settings-ui/Settings.UI/ViewModels/GeneralViewModel.cs

```diff
@@ -272,11 +274,20 @@ public bool IsAdmin
             }
         }
 
+        // Are we running a dev build? (Please note that we verify this in the code that gets the newest version from Github too.)
```

Comment:
![image](https://user-images.githubusercontent.com/26118718/221191932-b458157f-cb3f-49f0-8cff-b88307ef88b7.png)

I'll update this one.